### ISSUE TITLE
Some waypoints don't show up in map

### ISSUE THREAD

**Issue by MinhxNguyen7** [icssc/AntAlmanac#784]
The second waypoint is missing in this schedule. See schedule "map-error"

<img width="717" alt="image" src="https://github.com/icssc/AntAlmanac/assets/64875104/baf685b1-eac9-43cd-a5ad-01661a7c36d2">

**Comment by KevinWu098** [comment]
```typescript
    const validBuildingCodes = [...uniqueBuildingCodes].filter(
        (buildingCode) => buildingCatalogue[locationIds[buildingCode]] != null
    );
```

This is the reason why (from `Map.tsx`) ^^

The changes in #779, although confirmed to still work for all of the locations in map search, didn't take into account that `location_ids.ts` are the apparent source of truth for which ID in the list is the one we use. There may be more discrepancies still, but this is likely the source of the issue here

**Comment by KevinWu098** [comment]
Should we revert #779 in this instance? Or just push a patch for it?

**Comment by MinhxNguyen7** [comment]
Fixed by #785
